pkg/standalone-utils/contracts/LPOracleFactoryBase.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { AggregatorV3Interface } from "@chainlink/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import { ILPOracleFactoryBase } from "@balancer-labs/v3-interfaces/contracts/standalone-utils/ILPOracleFactoryBase.sol";
+import { ILPOracleBase } from "@balancer-labs/v3-interfaces/contracts/standalone-utils/ILPOracleBase.sol";
+import { IBasePool } from "@balancer-labs/v3-interfaces/contracts/vault/IBasePool.sol";
+import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
+
+import { InputHelpers } from "@balancer-labs/v3-solidity-utils/contracts/helpers/InputHelpers.sol";
+import { SingletonAuthentication } from "@balancer-labs/v3-vault/contracts/SingletonAuthentication.sol";
+
+abstract contract LPOracleFactoryBase is ILPOracleFactoryBase, SingletonAuthentication {
+    uint256 internal _oracleVersion;
+    mapping(IBasePool => ILPOracleBase) internal _oracles;
+    mapping(ILPOracleBase => bool) internal _isOracleFromFactory;
```

Comment:
```suggestion

    mapping(IBasePool pool => ILPOracleBase) oracle internal _oracles;
    mapping(ILPOracleBase oracle => bool creationFlag) internal _isOracleFromFactory;
```
Name the mapping parameters (even though they're kind of obvious). Would also separate the mappings from other variables.